pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -110,16 +110,10 @@ rustPlatform.buildRustPackage (finalAttrs: {
     owner = "zed-industries";
     repo = "zed";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-cnzaW0paORio8AONtVVGUZWG+HsWte12aWK2qvfj8oI=";
+    hash = "sha256-YkoIOBoR5hMt99D1bJ1yWLv7C/rY6VKC5J/7c5SMUFs=";
   };
 
   patches = [
-    # Zed uses cargo-install to install cargo-about during the script execution.
-    # We provide cargo-about ourselves and can skip this step.
-    # Until https://github.com/zed-industries/zed/issues/19971 is fixed,
-    # we also skip any crate for which the license cannot be determined.
-    ./0001-generate-licenses.patch
```

Comment:
Perhaps also re-number the remaining patch?

Reply:
Note that the remaining patch can also be removed once staging-next has landed in master